apps/mark/frontend/src/pages/replace_election_screen.tsx

```diff
@@ -35,33 +30,20 @@ export function ReplaceElectionScreen({
   appPrecinct,
   ballotsPrintedCount,
   electionDefinition,
-  getElectionDefinitionFromCard,
+  electionDefinitionFromCard,
   machineConfig,
   screenReader,
   unconfigure,
 }: Props): JSX.Element {
   const { election, electionHash } = electionDefinition;
-  const makeCancelable = useCancelablePromise();
-  const [cardElectionDefinition, setCardElectionDefinition] =
-    useState<ElectionDefinition>();
-
-  useEffect(() => {
-    void (async () => {
-      setCardElectionDefinition(
-        await makeCancelable(getElectionDefinitionFromCard())
```

Comment:
I no longer need to wrap the call to `getElectionDefinitionFromCard` in a `makeCancelable` because the `AppRoot` component is always mounted, unlike the `ReplaceElectionScreen` component, which could unmount in the middle of this request if the election manager card was removed